### ISSUE TITLE
File conflict icon missing - web interface

### ISSUE THREAD

**Issue by haukman** [owncloud/core#30089]
### Steps to reproduce
1. Use the WebUI to upload a pdf to a folder
2. Upload the same pdf again to the same folder

### Expected behaviour

The Pop-Up Window should show the PDF-Icon on both sides.

### Actual behaviour
This is what it looks like:

![grafik](https://user-images.githubusercontent.com/25486818/34774617-4bd66f10-f610-11e7-93b5-d0f02c5ca33c.png)

Easy to reproduce.

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)
10.0.4
**

**Comment by haukman** [comment]
@PVince81

**Comment by DeepDiver1975** [comment]
log file please

**Comment by haukman** [comment]
Can´t see anything related to this in the logs:
```

{"reqId":"JxjQmAqHxc1T5ZAyQv3a","level":3,"time":"2018-01-10T13:31:10+00:00","remoteAddr":"::1","user":"--","app":"PHP","method":"GET","url":"\/index.php","message":"Undefined variable: user at \/opt\/owncloud\/lib\/public\/Util.php#193"}
{"reqId":"XaFwEnVufJTI7wpVGpAw","level":2,"time":"2018-01-10T13:31:15+00:00","remoteAddr":"91.45.209.38","user":"--","app":"core","method":"GET","url":"\/","message":"Trusted domain error. \"91.45.209.38\" tried to access using \"docker.oc.solidgear.es:11312\" as host."}
{"reqId":"dnxWvJVCjxJKdr6NPV3j","level":2,"time":"2018-01-10T13:31:16+00:00","remoteAddr":"91.45.209.38","user":"--","app":"core","method":"GET","url":"\/index.php\/core\/js\/oc.js?v=e013adfbcde44eac369f6fa6233ca4f3","message":"Trusted domain error. \"91.45.209.38\" tried to access using \"docker.oc.solidgear.es:11312\" as host."}
{"reqId":"J26gw9utHwEUL96zskP7","level":2,"time":"2018-01-10T13:31:17+00:00","remoteAddr":"91.45.209.38","user":"--","app":"core","method":"GET","url":"\/cron.php","message":"Trusted domain error. \"91.45.209.38\" tried to access using \"docker.oc.solidgear.es:11312\" as host."}
{"reqId":"2LOLgoAdQh1OkIvCtKv9","level":2,"time":"2018-01-10T13:32:03+00:00","remoteAddr":"91.45.209.38","user":"--","app":"core","method":"POST","url":"\/index.php\/login","message":"Login failed: 'admin' (Remote IP: '91.45.209.38')"}
{"reqId":"Fm3DyCN3bChYkbIlxIC6","level":3,"time":"2018-01-10T13:32:09+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/apps\/files\/","message":"include(layout.noscript.warning.php): failed to open stream: No such file or directory at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"Fm3DyCN3bChYkbIlxIC6","level":3,"time":"2018-01-10T13:32:09+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/apps\/files\/","message":"include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='\/opt\/owncloud\/apps') at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"STG4BR1Sy5q1cKGB2QNa","level":3,"time":"2018-01-10T13:33:41+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/personal","message":"include(layout.noscript.warning.php): failed to open stream: No such file or directory at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"STG4BR1Sy5q1cKGB2QNa","level":3,"time":"2018-01-10T13:33:41+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/personal","message":"include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='\/opt\/owncloud\/apps') at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"jhvYEGu1l2y1lUpDKVkm","level":3,"time":"2018-01-10T13:33:44+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=diagnostics","message":"include(layout.noscript.warning.php): failed to open stream: No such file or directory at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"jhvYEGu1l2y1lUpDKVkm","level":3,"time":"2018-01-10T13:33:44+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=diagnostics","message":"include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='\/opt\/owncloud\/apps') at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"XUdwpZWvA0eKOFixVRBs","level":3,"time":"2018-01-10T13:33:51+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=help","message":"include(layout.noscript.warning.php): failed to open stream: No such file or directory at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"XUdwpZWvA0eKOFixVRBs","level":3,"time":"2018-01-10T13:33:51+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=help","message":"include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='\/opt\/owncloud\/apps') at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"B0yIS5IAdiujqaJnOYvq","level":3,"time":"2018-01-10T13:33:57+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=general","message":"opendir(\/opt\/owncloud\/themes): failed to open dir: No such file or directory at \/opt\/owncloud\/apps\/templateeditor\/lib\/mailtemplate.php#155"}
{"reqId":"B0yIS5IAdiujqaJnOYvq","level":3,"time":"2018-01-10T13:33:57+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=general","message":"include(layout.noscript.warning.php): failed to open stream: No such file or directory at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
{"reqId":"B0yIS5IAdiujqaJnOYvq","level":3,"time":"2018-01-10T13:33:57+00:00","remoteAddr":"91.45.209.38","user":"admin","app":"PHP","method":"GET","url":"\/index.php\/settings\/admin?sectionid=general","message":"include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='\/opt\/owncloud\/apps') at \/opt\/owncloud\/apps\/theme-enterprise\/core\/templates\/layout.user.php#36"}
```

**Comment by DeepDiver1975** [comment]
reproduced - looks like if no preview can be generated the fallback to the mime icon is not working in the javascript side of the conflict dialog.

this needs a js developer @felixheidecke

**Comment by DeepDiver1975** [comment]
![bildschirmfoto von 2018-01-10 16-26-14](https://user-images.githubusercontent.com/1005065/34780382-4a9c0e08-f623-11e7-91af-60d479ffd6d8.png)

**Comment by PVince81** [comment]
I think @VicDeo worked on previews lately when fixing something about supported mime types, so I'd rather ask him to have a look as he's more familiar

**Comment by phil-davis** [comment]
This is the sort of thing where you could have a million UI tests! There are so many buttons, fields, icons in a UI. Each of them (should) be there for a real reason. So if any go wrong it is a real regression. Any could go wrong with a range of combinations of file type, text input (unicode chars, special chars...),...

**Comment by VicDeo** [comment]
>you could have a million UI tests 

...and slow down your build by  2-3 hours more in  case your environment is capable to run 100 UI tests  per second.
Is it affordable price for a minor UI glitch.  :)

**Comment by phil-davis** [comment]
Yes, this is the problem with UI tests - they are slllllooooowwww. To get decent turnaround a whole farm of test infrastructure is needed to run 20 or 30 parallel scenarios.

**Comment by DeepDiver1975** [comment]
ok - let's agree to simply fix it - ok?

**Comment by VicDeo** [comment]
Fixed in master/stable10  with https://github.com/owncloud/core/pull/30123 and  https://github.com/owncloud/core/pull/30125
To be shipped with 10.0.6
